docs/documentation-styleguide.md

```diff
@@ -0,0 +1,185 @@
+# Carbon Documentation Styleguide
+## Follow these guidelines when contributing documentation to the Carbon repository
+
+### Contents
+
+[Introduction](#introduction)
+
+[What We Document](#what-we-document)
+* [User Facing Documentation](#user-facing-documentation)
+* [Release Notes and Changelog](#release-notes-and-changelog)
+* [Request For Change (RFC)](#request-for-change)
+* [README files](#readme-files)
+* [Setup and Getting Started Guides](#setup-and-getting-started-guides)
+* [Styleguides](#styleguides)
+* [Other Internal Documentation](#other-internal-documentation)
+  * [New Joiners Guide](#new-joiners-guide)
+  * [Carbon Roadmap](#carbon-roadmap)
+  
+[Language and Grammar Style](#language-and-grammar-style)
+* [Jargon](#jargon)
+* [Abbreviations (including Acronyms)](#abbreviations-including-acronyms)
+* [Capitalisation](#capitalisation)
+
+[Text Formatting and Organisation](#text-formatting-and-organisation)
+* [Code in text](#code-in-text)
+  * [Code string example](#code-string-example)
+  * [Code block example](#code-block-example)
+* [Bullet points and numbered lists](#bullet-points-and-numbered-lists)
+* [Tables](#tables)
+
+[Use of Links](#use-of-links)
+* [Linking to other sites](#linking-to-other-sites)
+* [Linking to other documents](#linking-to-other-documents)
+
+[Use of Images](#use-of-images)
+
+
+### Introduction
+This Documentation styleguide describes what is documented in the Carbon repository and which format this  takes to enable you to contribute documentation in line with the current styles. It includes:
```

Comment:
```suggestion
This Documentation style guide describes what is documented in the Carbon repository and which format this takes to enable you to contribute documentation in line with the current styles. It includes:
```

Reply:
I couldn't see that extra space between "this" and "takes" as the word "takes" had wrapped onto the next line in VS Code. Thanks 👍

Reply:
Separated "styleguide" into two words "style guide" throughout, apart from where it quotes another document.